### DISCUSSION TITLE
PBC with position vectors as nodal features

### DISCUSSION THREAD

**Discussion by TheMightiestCarrot** [e3nn/e3nn#467] [Q&A] [answered]
Hi, in my project i want to use ASE neighbour list as demonstrated in this tutorial: 
[Point inputs with periodic boundary conditions](https://docs.e3nn.org/en/stable/guide/periodic_boundary_conditions.html)
However, i also want to add position vectors as nodal features, and i wonder whether this is even possible. In the tutorial (also in the phonosDos tutorial) there are only non-geometrical (scalar) features, so the exact positions of the nodes don't seem to matter. 
How do i add position vectors as nodal features in systems with PBC?

Thank you very much!

**Comment by blondegeek** [comment, marked as answer]
Note that position vectors will depend on your choice of origin. The way
that translation equivariance is handled in CNNs of any kind is by dealing
with relative distance vectors, to including the position vectors directly
will break equivariance.
